composer.json

```diff
@@ -64,7 +63,8 @@
 		"npm-asset/perfect-scrollbar": "0.6.16",
 		"npm-asset/textcomplete": "^0.18.2",
 		"npm-asset/typeahead.js": "^0.11.1",
-		"phpseclib/phpseclib": "^2.0"
+		"phpseclib/phpseclib": "^2.0",
```

Comment:
Please move the phpseclib and the new language detection dependency over the npm/browser assets so we can differ between frontend and backend dependencies (I forgot it last time tbh ;-) )

Reply:
I guess I have to manually do something after altering that file. I'm not sure, what I actually had to do.

Reply:
I sorted it :-)